oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/AuthenticationCallbackHandlerTest.java

```diff
@@ -380,6 +380,9 @@ void shouldRedirectToFrontendErrorPageIfUserInfoRequestIsUnsuccessful()
     @Test
     void shouldUpdateOrchSessionUsingClaimsFromUserInfoResponse()
             throws UnsuccessfulCredentialResponseException {
+
+        when(USER_INFO.getBooleanClaim("new_account")).thenReturn(true);
+
```

Comment:
Is this line making a difference? Looks like line 138 on this file has the same statement.

Reply:
Oh yes not sure why this got added